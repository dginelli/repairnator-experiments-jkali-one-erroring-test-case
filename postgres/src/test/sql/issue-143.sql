SELECT zdb_estimate_count('so_posts', '({"term": {"body": "java"}})');