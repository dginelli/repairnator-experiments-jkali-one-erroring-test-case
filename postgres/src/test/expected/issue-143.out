SELECT zdb_estimate_count('so_posts', '({"term": {"body": "java"}})');
 zdb_estimate_count 
--------------------
              13310
(1 row)

